src/dataset/domain/models/Dataset.ts

```diff
@@ -1,5 +1,10 @@
+import { DatasetField } from './DatasetField'
+import { License } from './License'
+
 export interface Dataset {
   id: string
   title: string
   version: string
+  summaryFields: DatasetField[]
+  license: License
 }
```

Comment:
I suggest adding all the Dataset model interfaces to the same file. I like seeing DatasetFields and License as part of Dataset. Also since there is no logic in this kind of files, it doesn't bother anyone to see a long long file. This is just a data structure, I think it makes sense to see the whole structure at once. What do you think?

```

export interface License {
   name: string
   shortDescription: string
   uri: string
   iconUrl?: string
 }

export interface DatasetField {
   title: string
   description: string
   value: string
 }

export interface Dataset {
   id: string
   title: string
   version: string
   summaryFields: DatasetField[]
   license: License
 }

```

Reply:
Thanks for the comments, Melina!  Those suggestions sound good to me, just have a question or two, will respond above 👍